### ISSUE TITLE
Default Color Schemes For Buttons Fail WCAG 2.1 AA Criteria

### ISSUE THREAD

**Issue by Jack-Merideth-Callibrity** [chakra-ui/chakra-ui#8111]
### Description

When using axe-core (or the axe DevTools extension), I would expect Chakra elements to not be flagged as having an issue, but instead got the Chakra Buttons with a default color scheme (blue) being marked as failing [WCAG 2.1 AA Criterion 1.4.3](https://www.w3.org/WAI/WCAG22/quickref/#contrast-minimum). Checking the Color Schemes on the documentation site, the following color schemes also fail this WCAG Criterion:

- Red
- Orange
- Green
- Teal
- Pink
- Linkedin
- Messenger
- Whatsapp
- Twitter
- Telegram

### Link to Reproduction

https://codesandbox.io/s/chakra-button-demo-gnpwgj-gnpwgj

### Steps to reproduce

Method One:
1. Go to [Chakra UI Button Documentation](https://chakra-ui.com/docs/components/button/usage)
2. Open Dev Pane
3. Trigger axe DevTools to do a scan.
4. Page through color contrast errors to verify the failing buttons

Method Two:
1. Go to [Chakra UI Button Documentation](https://chakra-ui.com/docs/components/button/usage)
2. Open Dev Pane
3. For each sample button in the colorScheme, pull the background and font color into a contrast checker (one cane be found here: https://webaim.org/resources/contrastchecker/)

### Chakra UI Version

^2.4.6

### Browser

Chrome, Firefox, Electron (via Cypress)

### Operating System

- [X] macOS
- [X] Windows
- [X] Linux

**Comment by segunadebayo** [comment]
Indeed, this is a design flaw in v2. I recommend customizing the components in userland to improve accessibility.

We've improved this in v3 by leveraging semantic tokens.

Thanks for understanding.
